packages/server/test/cache_helper.js

```diff
@@ -0,0 +1,19 @@
+const supportedConfigFiles = [
+  'cypress.json',
+  'cypress.config.js',
+]
+
+/**
+   * Since we load the cypress.json, cypress.e2e.config.js or cypress.component.config.js
+   * via `require`, we need to clear the `require.cache` before/after some tests
+   * to ensure we are not using a cached configuration file.
+   */
```

Comment:
```suggestion
/**
 * Since we load the cypress.json, cypress.e2e.config.js or cypress.component.config.js
 * via `require`, we need to clear the `require.cache` before/after some tests
 * to ensure we are not using a cached configuration file.
 */
```